lib/src/builder.dart

```diff
@@ -98,7 +99,11 @@ class GeneratorBuilder extends Builder {
 
 Stream<GeneratedOutput> _generate(LibraryElement unit,
     List<Generator> generators, BuildStep buildStep) async* {
-  for (var element in getElementsFromLibraryElement(unit)) {
+  var elements = safeIterate(getElementsFromLibraryElement(unit), (e, [_]) {
```

Comment:
We could. It would introduce a `package:build` import into `utils.darts` so we could do the logging, and we'd probably end up needing to pass through the `buildStep` or `inputId` which IMO degrades from the nice separation we have now.

My preference would be this layout, WDYT?